pkg/signals/signals.go

```diff
@@ -0,0 +1,20 @@
+package signals
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func CreateChannel() (chan os.Signal, func()) {
+	stopCh := make(chan os.Signal, 1)
+	signal.Notify(
+		stopCh,
+		os.Interrupt,
+		syscall.SIGTERM,
+		syscall.SIGINT)
+
+	return stopCh, func() {
+		close(stopCh)
+	}
+}
```

Comment:
follow-up: let's move this package to internal like we did with the http package.